types/scenegraph.d.ts

```diff
@@ -319,6 +435,15 @@ export class ImageFill {
      */
     scaleBehaviour: string;
 
+    // TODO: assetId: string (readonly?)
+    /**
+     * (**Since**: XD 29)
+     *
+     * A unique identifier for the image asset used by this ImageFill. May be shared by other ImageFills, including those with different cropping, size,
+     rotation, or mirroring. If identical images are imported into XD from separate sources, they may have different `assetId`s however.
```

Comment:
Formatting. This is a admittedly a bit pedantic.

```suggestion
     * rotation, or mirroring. If identical images are imported into XD from separate sources, they may have different `assetId`s however.
```